Models/User.cs

```diff
@@ -49,6 +49,13 @@ public User()
 
         public string ProfileUrl { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the user profile is public.
+        /// </summary>
+        /// <value>
+        ///   <c>true</c> if this instance is public; otherwise, <c>false</c>.
+        /// </value>
+        public bool IsPublic { get; set; }
```

Comment:
We should set the default to false. 
We want this to be an opt-in feature.

Reply:
The default value is false

Reply:
Right, I keep forgetting this. My bad......